package/debian/debian/changelog

```diff
@@ -1,5 +1,5 @@
-gittown (3.1.0) RELEASED; urgency=medium
+gittown (4.2.1) RELEASED; urgency=low
 
   * See https://github.com/Originate/git-town/releases for a complete changelog
 
- -- allonsy <alec.snyder@originate.com>  Fri, 05 Aug 2016 18:58:41 +0000
+ -- allonsy <alec.snyder@originate.com> Tue, 29 Aug 2017 16:17:35 -0700
```

Comment:
Can this file be removed? It appears a new one is created in `make_deb.sh`

Reply:
Sure